packages/mermaid/src/dagre-wrapper/markers.js

```diff
@@ -2,6 +2,59 @@
 
 import { log } from '../logger.js';
 
+const getSvgParent = (elem) => {
+  let container = elem;
+
+  // the intent here is to find the first parent element that is NOT part of the SVG element
+  // I know there has to be a better way, but could not find one that worked
+  // tried using checking if elem was instanceof SVGGraphicsElement or SVGElement, but it failed to detect correctly
+  if (container._groups) {
+    container = container._groups[0][0];
+  }
+  if (container.tagName.toLowerCase() === 'g') {
+    container = container.parentElement;
+  }
+  if (container.localName.toLowerCase() === 'svg') {
+    container = container.parentElement;
+  }
+  return container;
+};
+
+/**
+ * Finds the parent background color of an SVG element.
+ *
+ * Used to make a "hollow" arrowhead for an arrow.
+ *
+ * We can't use a transparent fill,
+ * because the arrow line is behind the arrowhead
+ * (ideally we'd stop drawing the line behind the arrowhead,
+ *  but this is pretty complicated to do).
+ *
+ * **Limitations**:
+ * - If the parent background color is a partial transparency,
+ *   the arrowhead will also be partially transparent.
+ * - More complicated backgrounds like pictures/animations won't work.
+ * @param elem
```

Comment:
Can you add a short description for the `elem` parameter (it's mainly to make `eslint`/`tsdoc` lint checkers happy)?

Maybe something like:

```suggestion
 * @param elem The SVG element.
```